operating/manage-user-access/manage-role-based-access-control-3630.adoc

```diff
@@ -0,0 +1,62 @@
+= Managing role-based access control in {product-title} 3.63.0 and newer
+include::modules/common-attributes.adoc[]
+:context: manage-role-based-access-control
+
+toc::[]
+
+[role="_abstract"]
+{product-title} (RHACS) comes with role-based access control (RBAC) that you can use to configure roles and grant various levels of access to {product-title} to different users.
+
+[IMPORTANT]
+====
+If you are using {product-title} 3.0.62 or older see xref:../../operating/manage-user-access/manage-role-based-access-control-3062.adoc#manage-rbac-3062[Managing access control in {product-title} 3.0.62 and older].
+====
+
+{product-title} 3.63.0 include scoped access control feature which enables you to configure fine-grained and specific sets of permissions that define how a given {product-title} user, or group of users, can interact with {product-title}, which resources they can access, and which actions they can perform.
+
+* _Roles_ are a collection of permission sets and access scopes. You can assign roles to users and groups by specifying rules. You can configure these rules when you configure an authentication provider.
+There are two types of roles in {product-title}:
+** System roles that are created by Red Hat and cannot be changed.
+** Custom roles, which {product-title} administrators can create and change at any time.
++
+[NOTE]
+====
+* If you assign multiple roles for a user, they get access to the combined permissions of the assigned roles.
+* If you have users assigned to a custom role, and you delete that role, all associated users transfer to the minimum access role that you have configured.
+====
+
+* _Permission sets_ are a set of permissions that define what actions a role can perform on a given resource. _Resources_ are the functionalities of {product-title} for which you can set view (`read`) and modify (`write`) permissions. There are two types of permission sets in {product-title}:
+** System permission sets, which are created by Red Hat and cannot be changed.
+** Custom permission sets, which {product-title} administrators can create and change at any time.
+
+* _Access scopes_ are a set of Kubernetes and {ocp} resources that users can access. For example, you can define an access scope which only allows users to access information about pods in a given project. There are two types of access scopes in {product-title}:
+** System access scopes, which are created by Red Hat and cannot be changed.
+** Custom access scopes, which {product-title} administrators can create and change at any time.
+
+include::modules/rbac-system-roles-3630.adoc[leveloffset=+1]
+
+include::modules/view-system-roles-permission-scope.adoc[leveloffset=+2]
+
+include::modules/create-a-custom-role-3630.adoc[leveloffset=+2]
+
+[role="additional-resources"]
+.Additional resources
+* xref:../../operating/manage-user-access/manage-role-based-access-control.adoc#create-a-custom-permission-set_manage-role-based-access-control[Creating a custom permission set]
```

Comment:
Please double-check these xrefs as they are not working currently. Perhaps the target files are not merged yet?